app/assets/javascripts/controllers/credentials/credentials_controller.js

```diff
@@ -1,55 +1,57 @@
-ManageIQ.angular.app.controller('CredentialsController', ['$scope', function($scope) {
-  var init = function() {
-    $scope.bChangeStoredPassword = undefined;
-    $scope.bCancelPasswordChange = undefined;
-
-    $scope.$on('resetClicked', function(_e) {
-      $scope.resetClicked();
-    });
-
-    $scope.$on('setNewRecord', function(_event, args) {
-      $scope.newRecord = args ? args.newRecord : true;
-    });
-
-    $scope.$on('setUserId', function(_event, args) {
-      if (args) {
-        $scope.modelCopy[args.userIdName] = args.userIdValue;
-      }
-    });
-
-    if ($scope.formId == 'new') {
-      $scope.newRecord = true;
-    } else {
-      $scope.newRecord = false;
-      $scope.bChangeStoredPassword = false;
-      $scope.bCancelPasswordChange = false;
-    }
-  };
+ManageIQ.angular.app.controller('CredentialsController', ['$scope', '$attrs', function($scope, $attrs) {
```

Comment:
So.. this `$attrs` thing breaks all users of CredentailsController which don't provide `vmScope`.

So GCE is broken.. creating a PR..